core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Util.java

```diff
@@ -329,14 +329,14 @@ static Long expiresAtMillis(String token) {
       return null;
     }
 
-    String[] parts = token.split("\\.");
-    if (parts.length != 3) {
+    List<String> parts = Splitter.on('.').splitToList(token);
```

Comment:
nit: should we extract this to a `private static final Splitter DOT_SPLITTER`?

Reply:
As it is not used in multiple places (in the same file), I thought no need for an extra variable.